tests/test.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+import pybamm
+import pybamm_tea as TEA
+import matplotlib.pyplot as plt
+import matplotlib.patches as patches
+import pandas as pd
+
+class TestTEA(unittest.TestCase):
+    # load example model
+    def ExampleModel(self):
+        nco_input_data = {
+            "Electrolyte density [kg.m-3]": 1276,
+            "Negative electrode active material density [kg.m-3]": 2266,  # Graphite
+            "Positive electrode active material density [kg.m-3]": 4750,  # NCO
+        }
+        param_nco = pybamm.ParameterValues("Ecker2015")
+        param_nco.update(nco_input_data, check_already_exists=False)
+        tea_nco = TEA(param_nco)
+        return tea_nco
+    
+    # test stack energy densities dict
+    def test_stack_energy_densities(self):
+        tea_nco = self.ExampleModel()
+        assert isinstance(tea_nco.stack_energy_densities, dict), "stack_energy_densities is not a dict"
```

Comment:
check the unittest docs for proper assertions. e.g. here you should use `self.assertIsnstance`